react/react1/week1/todolist/src/components/Header.js

```diff
@@ -0,0 +1,24 @@
+import AddTodo from "./AddTodo";
+function Header({
+  children,
+  addTodo,
+  addItemText,
+  value,
+  startDate,
+  datepicker,
+}) {
+  return (
+    <div className="header">
+      {children}
```

Comment:
I would advise you to read up on component composition in React. You are actually allready using children here and therefore you can just put the AddToDo component in the parent component as you are allready doing here. This way you avoid having to pass all this props down (prop drilling) and just like in ToDoList component you are adding a h1 you can add the AddToDo component inside the Header component with children.